static/app/views/starfish/components/detailPanel.tsx

```diff
@@ -52,8 +75,35 @@ export default function Detail({children, detailKey, onClose, onOpen}: DetailPro
   }, [escapeKeyPressed]);
 
   return (
-    <SlideOverPanel collapsed={state.collapsed} ref={panelRef} onOpen={onOpen}>
+    <SlideOverPanel
+      slideDirection={slideDirection}
```

Comment:
Nit, but I think `direction` is the wrong word here. The panel _position_ and the panel _slide direction_ can be different (e.g., the panel is docked to the bottom, but slides in from the top). Since this is describing the position, I think this prop should be called `position` everywhere